templates/basic-ts/test/utils/jest.overrides.js

```diff
@@ -0,0 +1,6 @@
+let warn = console.warn
+
+console.warn = function (message) {
+    warn.apply(console, arguments) // keep default behaviour
+    throw (message instanceof Error ? message : new Error(message))
+}
```

Comment:
the missing new line should have failed the linting with prettier 🤔

Reply:
If I get it right at the moment no linting test step is defined, there's a `lint:fix` inside the Bash script test though: https://github.com/probot/create-probot-app/blob/7c905fabd8fc37141127f687036ab29bfc1d2d27/script/test#L18